module/x/gravity/types/key.go

```diff
@@ -58,108 +58,190 @@ const (
 	// LastUnBondingBlockHeightKey indexes the last validator unbonding block height
 	LastUnBondingBlockHeightKey
 
+	// LastObservedSignerSetKey index the last observed signer set
 	LastObservedSignerSetKey
 
-	// EthereumHeightVoteKey indexes the latest heights observed by each validator
-	EthereumHeightVoteKey
+	// EVMHeightVoteKey indexes the latest heights observed by each validator
+	EVMHeightVoteKey
 )
 
 ////////////////////
 // Key Delegation //
 ////////////////////
 
+// Uint32ToBigEndian - marshals uint64 to a bigendian byte slice so it can be sorted
+func Uint32ToBigEndian(i uint32) []byte {
+	b := make([]byte, 8)
+	binary.BigEndian.PutUint32(b, i)
+	return b
+}
+
+// BigEndianToUint32 returns a uint32 from big endian encoded bytes. If encoding
+// is empty, zero is returned.
+func BigEndianToUint32(bz []byte) uint32 {
+	if len(bz) == 0 {
+		return 0
+	}
+
+	return binary.BigEndian.Uint32(bz)
+}
+
 // MakeOrchestratorValidatorAddressKey returns the following key format
 // prefix
-// [0xe8][cosmos1ahx7f8wyertuus9r20284ej0asrs085case3kn]
+// [0xe8][0001][cosmos1ahx7f8wyertuus9r20284ej0asrs085case3kn]
```

Comment:
I don't think we're adding any bytes here...

Reply:
Yes, relic from before, when I had chain specific delegations

Reply:
removed